index.js

```diff
@@ -79,18 +79,14 @@ module.exports = {
     "no-trailing-spaces": 1,
     "no-useless-constructor": 1,
     "one-var": [ 1, { "uninitialized": "always", "initialized": "never" } ],
-
-    /* "object-curly-spacing": [ 2, "never" ], */
-
+    "object-curly-spacing": [ 1, "never" ],
```

Comment:
Thanks for the comments. Yes it is more constricting, but isn't that the point of eslint?

For curly-spacing: this is a _semantic_ style decision that I think we've already decided on. When skimming, I can tell that it is an object and not a destructured one. If there is padding between the curlies, I know it must be destructured.

For parens: using our editors to better color parens is just a more pragmatic solution. I still prefer the looks of spacing within parens, but when hacking away, I'm too lazy to press the space bar twice. Someday I hope editors will allow me to specify rules to auto-pad parens in some situations and not in others, but we aren't there yet. Maybe https://github.com/jlongster/prettier will become configurable-enough for us to use soon.

It is important to remember that the point of these rules is to better enable us to write code that 1) We don't hate looking at, and 2) We are better able to skim.